docs/quickstart/launch.md

```diff
@@ -18,10 +18,7 @@ _The quickstart uses Docker to bring up the lakeFS container, pre-populate it wi
 Launch the lakeFS container:
 
 ```bash
-docker run --name lakefs --pull always \
-             --rm --publish 8000:8000 \
-             treeverse/lakefs:latest \
-             run --local-settings
+docker run --name lakefs --pull always --rm --publish 8000:8000 treeverse/lakefs:latest run --local-settings
```

Comment:
![CleanShot_2023-07-20_at_11 25 18](https://github.com/treeverse/lakeFS/assets/3671582/097279be-e433-452b-b4f7-ec55cb0b40b1)

I realise this bypasses the linebreak incompatibility issue for Windows, but it makes it less readable and clear what's going on. 

Maybe we use tabs (like we do elsewhere in docs) so there's a linux/unix/mac tab, and then a windows tab?